test/integration/deep/opaqueports/assertions.go

```diff
@@ -41,13 +39,12 @@ func hasNoOutboundHTTPRequest(metrics, ns string) error {
 func hasOutboundHTTPRequestWithTLS(metrics, ns string) error {
 	m := prommatch.NewMatcher("request_total", prommatch.Labels{
 		"direction":          prommatch.Equals("outbound"),
-		"target_addr":        prommatch.Like(addrRE),
-		"target_ip":          prommatch.Like(ipRE),
 		"tls":                prommatch.Equals("true"),
 		"server_id":          prommatch.Equals(fmt.Sprintf("default.%s.serviceaccount.identity.linkerd.cluster.local", ns)),
 		"dst_namespace":      prommatch.Equals(ns),
 		"dst_serviceaccount": prommatch.Equals("default"),
 	},
+		prommatch.TargetAddrLabels(),
```

Comment:
This formatting is a little surprising... could we put `prommatch.Labels{` on a newline so that it gets indented to match the rest of the fields?

There are a few instances of this in this field; if you could correct them all that'd be great!

Reply:
RIght, it looks better if `prommatch.Labels` is in a new line. Done in https://github.com/linkerd/linkerd2/pull/8674/commits/2cf50f87a2681b826c1874d23212bfdda12d66eb